Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Rapid7_Enrichment.yml

```
@@ -0,0 +1,1004 @@
+id: Cortex ASM - Rapid7 Enrichment
+version: -1
+name: Cortex ASM - Rapid7 Enrichment
+description: Given the IP address this playbook enriches Rapid7 InsightVM (Nexpose) information relevant to ASM alerts.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 806c021a-3cbc-4873-8462-6f0d82ea9589
+    type: start
+    task:
+      id: 806c021a-3cbc-4873-8462-6f0d82ea9589
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "14"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 460,
+          "y": -220
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: ef3d2596-6f23-4e7d-8a37-15b156dbc4e8
+    type: playbook
+    task:
+      id: ef3d2596-6f23-4e7d-8a37-15b156dbc4e8
+      version: -1
+      name: Rapid7 - Nexpose - Enrichment
+      description: Given an IP address, this playbook searches Rapid7 Nexpose assets for an asset with the associated IP address to retrieve asset information and proceeds to return all associated tags with it.
+      playbookName: Rapid7 - Nexpose - Enrichment
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      IP:
+        complex:
+          root: inputs.IPAddress
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 680,
+          "y": 220
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```

https://qadium.atlassian.net/wiki/spaces/PM/pages/2383872027/XSOAR+XSIAM+Dev+Standards#Check-if-integration-enabled

```suggestion
    skipunavailable: true
```

---

Resolved in b2ffd19.